scania-digital-design-system/tegel
# fix(card): thumbnail slot

Removed header-img and img-alt and opened up slot card-thumbnail instead

BREAKING CHANGE: Instead of header-img, use the slot card-thumbnail

**Describe pull-request**  
Removed the header-img and ing-alt props and opened up slot instead.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1779

**How to test**  
1. Go to storybook
2. Check in Card
3. Check the API and try out the card-thumbnail slot.

**Suggested test steps**
